contracts/DCNT721A.sol

```diff
@@ -65,9 +85,14 @@ contract DCNT721A is ERC721A, Initializable, Ownable, Splits {
     MAX_TOKENS = _editionConfig.maxTokens;
     tokenPrice = _editionConfig.tokenPrice;
     maxTokenPurchase = _editionConfig.maxTokenPurchase;
+    saleStart = _editionConfig.saleStart;
     royaltyBPS = _editionConfig.royaltyBPS;
-    splitMain = _splitMain;
+    adjustableCap = _editionConfig.adjustableCap;
```

Comment:
```suggestion
    adjustableCap = _editionConfig.isAdjustableCap;
```

Was wondering if this is `boolean` or `uint256` I think changing name to `isX` helps to easily convey boolean nature but this is personal preference.

Reply:
this is a wider refactor than being suggested here, but i do agree with this sentiment. didn't personally name this var, but will keep in mind for future :)